src/llmq/quorums.cpp

```diff
@@ -164,7 +164,7 @@ int CQuorum::GetMemberIndex(const uint256& proTxHash) const
     return -1;
 }
 
-void CQuorum::WriteContributions(CEvoDB& evoDb) const
+void CQuorum::WriteContributions(const std::unique_ptr<CDBWrapper>& db) const
```

Comment:
nit: no need to pass here unique_ptr, a bit too much. `CDWrapper&` works just fine